HW3/KachanovskiAnton/task_2.py

```diff
@@ -0,0 +1,19 @@
+four_digit_natural_number = input("Enter a four-digit natural number: ")
+
+if len(four_digit_natural_number) != 4:
+    print("Please enter a valid four-digit natural number.")
+    four_digit_natural_number = input("Enter a four-digit natural number: ")
+else:
+    digits = [int(digit) for digit in four_digit_natural_number]
+
```

Comment:
look at this implementation
number = int(input("Enter four digit number:  "))
num_1 = number // 1000
num_2 = (number // 100) % 10
num_3 = (number // 10) % 10
num_4 = number % 10
product = num_1 * num_2 * num_3 * num_4
print(product)